### PR TITLE
removed library of play-services-ads

### PR DESCRIPTION
**Pull request by anshumansworld**
Removed google-play-services-ads library that was being used to get advertisingId.
Now we have implemented our own class to get AdvertisingId

### CODE REVIEW

File: library.demo/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
-    <meta-data
+    <!--<meta-data
         android:name="com.google.android.gms.version"
-        android:value="@integer/google_play_services_version" />
+        android:value="@integer/google_play_services_version" />-->
```
Comment (cerberillo) [inline library.demo/src/main/AndroidManifest.xml]: Please remove this from the manifest

File: library/src/main/java/net/pubnative/library/advertising/AdvertisingIdClient.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
+package net.pubnative.library.advertising;
```
Comment (cerberillo) [inline library/src/main/java/net/pubnative/library/advertising/AdvertisingIdClient.java]: can we move this to the utils package?

**Comment by cerberillo** [comment]
So far good, whenever fixed the comments, you can merge :+1:

File: library/src/main/java/net/pubnative/library/advertising/AdvertisingIdClient.java (hunk 2 above)
Comment (anshumansworld) [inline library/src/main/java/net/pubnative/library/advertising/AdvertisingIdClient.java]: :+1:

File: library.demo/src/main/AndroidManifest.xml (hunk 1 above)
Comment (anshumansworld) [inline library.demo/src/main/AndroidManifest.xml]: :+1:

**Pull request merged**